core/src/main/java/org/apache/cxf/common/jaxb/JAXBContextCache.java

```diff
@@ -336,7 +336,13 @@ private static JAXBContext createContext(final Set<Class<?>> classes,
         try {
             ctx = AccessController.doPrivileged(new PrivilegedExceptionAction<JAXBContext>() {
                 public JAXBContext run() throws Exception {
-                    return JAXBContext.newInstance(classes.toArray(new Class<?>[0]), map);
+                    //This is a workaround for CXF-8675
+                    Class factoryClass = ClassLoaderUtils.loadClass("org.glassfish.jaxb.runtime.v2.ContextFactory",
```

Comment:
@reta You are right. I've already move this workaround in jaxb/JAXBUtils, and other places will call in this new added `createContext` method.  Please review.
BTW, this jaxb-api issue is already fixed in upstream https://github.com/eclipse-ee4j/jaxb-api/pull/225. But it will take some time to have a 3.x release.